pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -0,0 +1,115 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchpatch
+, fetchurl
+, pkg-config
+, cmake
+, python3
+, libpng
+, zlib
+, eigen
+, protobuf
+, howard-hinnant-date
+, nlohmann_json
+, boost
+, oneDNN
+, gtest
+}:
+
+let
+  abseil = fetchurl {
+    url = "https://github.com/abseil/abseil-cpp/archive/refs/tags/20211102.0.zip";
+    sha1 = "ce61532df974d00025b1220408ce1c900d81baf2";
```

Comment:
Please change this to a `sha256` hash instead.

Reply:
I copied the sha1 from CMake file, but I switched to sha256.